ca/certificate-authority_test.go

```diff
@@ -165,13 +166,14 @@ type mockSA struct {
 	certificate core.Certificate
 }
 
-func (m *mockSA) AddCertificate(der []byte, _ int64) (string, error) {
+func (m *mockSA) AddCertificate(ctx context.Context, der []byte, _ int64) (string, error) {
 	m.certificate.DER = der
 	return "", nil
 }
 
 var caKey crypto.Signer
 var caCert *x509.Certificate
+var contxt = context.Background()
```

Comment:
Alternately: It looks like you're avoiding using the name `ctx`, possibly because it's used as an arg to AddCertificate. But I think it would be fine to use `ctx`. You can also use `_` for AddCertificate's param.